build_rules.gradle

```diff
@@ -363,6 +366,18 @@ ext.applyJavaNature = {
       force library.java.values()
     }
   }
+
+  def aptGeneratedMain = "${project.buildDir}/generated/source/apt/main"
+  def aptGeneratedTest = "${project.buildDir}/generated/source/apt/test"
+  idea {
+    module {
+      sourceDirs += file(aptGeneratedMain)
+      generatedSourceDirs += file(aptGeneratedMain)
+
+      sourceDirs += file(aptGeneratedTest)
```

Comment:
Also, for eclipse it seems like we are using the net.lgt-apt plugin, so it might be worthwhile to also use the idea version as well.

`sourceDirs` ->  `testSourceDirs`?

Reply:
Done